app/rules/BucketRuleManager.scala

```diff
@@ -12,11 +12,12 @@ import play.api.libs.json.Reads
 
 import model.{RegexRule, RuleResource}
 import play.api.Logging
+import com.amazonaws.services.s3.model.PutObjectResult
 
 class BucketRuleManager(s3: AmazonS3, bucketName: String) extends Logging {
     private val RULES_KEY = "rules/typerighter-rules.json"
 
-    def putRules(ruleResource: RuleResource): Either[Exception, Date] = {
+    def putRules(ruleResource: RuleResource): Either[Exception, PutObjectResult] = {
```

Comment:
It looks like we're not making use of the result of this method elsewhere, and leaking Amazon request internals to other services might be worth avoiding if that is the case – recommend just returning Unit, as this is a side-effecting function with (I think!) nothing meaningful to return.